nss-day-cohort-47/holiday-road-salt-lake-skaters
# added switch event operators

# Description
Added initial event listeners in an event listener module for all change and click events. 

Fixes # (issue)

## Type of change


# Testing Instructions
Load homepage. Click on buttons or select a dropdown. A message should appear in your console logging your action.

# Checklist:
